packages/vue/src/examples/pages/checkout/_internal/PersonalDetails.vue

```diff
@@ -103,13 +101,16 @@ export default {
     SfCheckbox,
     SfButton,
     SfHeading,
-    SfModal,
     SfCharacteristic
   },
   props: {
-    order: {
+    value: {
       type: Object,
       default: () => ({})
+    },
+    buttonName: {
+      type: String,
+      default: ""
     }
   },
   data() {
```

Comment:
@martaradziszewska when you mount this page again you start with empty form :) you must fill this internal data on mount